include/SH3/system/input_config.hpp

```diff
@@ -0,0 +1,31 @@
+/** @file
+ *  Defines configuration variables for the @ref sh3::system::input_system.
+ */
+#ifndef SH3_SYSTEM_INPUT_CONFIG_HPP_INCLUDED
+#define SH3_SYSTEM_INPUT_CONFIG_HPP_INCLUDED
+
+/**
+ *  Whether the @ref sh3::system::input_system should provide timing support for inputs.
+ *  
+ *  With this enabled, the @ref sh3::system::input_system keeps track what fraction of a frame the actions were active for.
+ *  
+ * @note Doesn't work right with SDL timestamps.
+ *       I think the event times are not when the input *happened*, but when the event-loop *received* them, which is not useful to us.
+ */
+#ifdef DOXYGEN
+#define INPUT_PROVIDE_TIMING 0
```

Comment:
> You could let the character move for distance * NormalizedTime(PRESSED)

Ahh, that's a lot clearer. I'll have to look into the movement system of both the games to see what percentage of a unit the character moves per frame (as there could be sneaking animations if you use a joystick controller). 

> For single-player we can remedy this effect by bounding the delta-time on updates, so the game itself slows down if a desired frametime cannot be reached.

Sounds good.